wbooks/src/app/screens/bookDetail/index.js

```diff
@@ -1,28 +1,49 @@
 import React, {Component} from 'react';
-import booksJson from '../../../../resources/books.json';
 import {Link} from 'react-router-dom';
 import './style.css';
 import PropTypes from 'prop-types';
 import Comments from './components/Comments';
 import NotFoundPage from '../../components/NotFoundPage';
+import UnauthorizedPage from '../../components/UnauthorizedPage';
 import { DASHBOARD, BOOKS } from '../../../config/routes';
+import * as bookService from '../../../services/bookService';
 
 class BookDetail extends Component {
 
   state = {book: null,
-    comments: [{id: 1, user: 'PERPE', date: '20/43', text:'safamdkfmksgdkj g kjf kjfk jfkj fkjkdjfjksperoowo opwporo owpkr eokr o'},{id: 2, user: 'PERPE', date: '20/43', text:'safamdkfmksgdkj g kjf kjfk jfkj fkjkdjfjksperoowo opwporo owpkr eokr 2'}]};
+    comments: [{id: 1, user: 'PERPE', date: '20/43', text:'safamdkfmksgdkj g kjf kjfk jfkj fkjkdjfjksperoowo opwporo owpkr eokr o'},{id: 2, user: 'PERPE', date: '20/43', text:'safamdkfmksgdkj g kjf kjfk jfkj fkjkdjfjksperoowo opwporo owpkr eokr 2'}],
+    relatedBooks: []
+  };
 
   componentWillMount(){
-    this.setState({book: this.getBookInfo(parseInt(this.props.match.params.id, 10))});
+    this.getBookInfo(this.props.match.params.id);
   }
 
 
   getBookInfo(bookId){
-      return booksJson.find((book) => book.id === bookId);
+    bookService.getBook(bookId, localStorage.getItem('accessToken'), this.onGetBookSuccess, this.onGetBookFailure);
```

Comment:
you should set the header with the token in `config/api` methods  ==> `get` and `post`